cmd/jimmsrv/main.go

```diff
@@ -207,30 +207,36 @@ func start(ctx context.Context, s *service.Service) error {
 		Handler:           jimmsvc,
 		ReadHeaderTimeout: time.Second * 5,
 	}
+
+	maxConccurentConncetions, _ := strconv.Atoi(os.Getenv("JIMM_SSH_MAX_CONCURRENT_CONNECTIONS"))
+
+	sshServer, err := ssh.NewJumpServer(ctx, ssh.Config{
+		Port:                     os.Getenv("JIMM_SSH_PORT"),
+		HostKey:                  []byte(hostKeyRaw),
+		MaxConcurrentConnections: maxConccurentConncetions,
+	}, jimmsvc.JIMM().SSHManager())
+
 	s.OnShutdown(func() {
 		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 		defer cancel()
 
-		zapctx.Warn(ctx, "server shutdown triggered")
+		zapctx.Warn(ctx, "HTTP server shutdown triggered")
 		err = httpsrv.Shutdown(ctx)
 		if err != nil {
-			zapctx.Error(ctx, "failed to shutdown server gracefully", zap.Error(err))
+			zapctx.Error(ctx, "failed to shutdown HTTP server gracefully", zap.Error(err))
+		}
+
+		zapctx.Warn(ctx, "SSH server shutdown triggered")
+		err = sshServer.Shutdown(ctx)
+		if err != nil {
+			zapctx.Error(ctx, "failed to shutdown SSH server gracefully", zap.Error(err))
 		}
+
 		jimmsvc.Cleanup()
 	})
 	s.Go(httpsrv.ListenAndServe)
-	zapctx.Info(ctx, "Successfully started JIMM server")
-	maxConccurentConncetions, _ := strconv.Atoi(os.Getenv("JIMM_SSH_MAX_CONCURRENT_CONNECTIONS"))
-
-	sshServer, err := ssh.NewJumpServer(ctx, ssh.Config{
-		Port:                     os.Getenv("JIMM_SSH_PORT"),
-		HostKey:                  []byte(hostKeyRaw),
-		MaxConcurrentConnections: maxConccurentConncetions,
-	}, jimmsvc.JIMM().SSHManager())
-	if err != nil {
-		return err
-	}
+	zapctx.Info(ctx, "Successfully started JIMM HTTP server")
 	s.Go(sshServer.ListenAndServe)
-	zapctx.Info(ctx, "Successfully started JIMM ssh server")
+	zapctx.Info(ctx, "Successfully started JIMM SSH server")
```

Comment:
Just "Started", you don't know if it's successful

Reply:
I mean sure, but this wasn't the point of this PR.